scripts/tools-setup.js

```diff
@@ -0,0 +1,18 @@
+const liquibase = require("liquibase");
+const path = require("path");
+
+// see README > "Integration tests" for why this awful code exists
+(async () => await liquibase({
+    liquibase: path.join(__dirname, "../node_modules/liquibase/lib/liquibase-4.0.0/liquibase"),
```

Comment:
we may want to dynamically get the path as I am thinking let's say we were to change the liquidbase version we use, this script would fail

Reply:
Resolved: file deleted with Tanio's blessing https://github.com/dvsa/cvs-tsk-update-store/pull/1/commits/e8fe927eb34091e84369f214815a38672a2b7f48